src/main/java/com/locallampoon/fiveh/ui/ConsolePanel.java

```diff
@@ -82,9 +81,11 @@ public void keyPressed(KeyEvent e) {
     @Override
     public void keyReleased(KeyEvent e) {
         int keyCode = e.getKeyCode();
-        if (keyCode == ENTER_KEY) {
-            // reset caret position
-            textArea.setCaretPosition(textArea.getCaretPosition() - 1);
+        if (keyCode == Key.ENTER.getKey()) {
```

Comment:
non-blocking. Any reason why to initialize keyCode variable on line 83 but not on line 73?

Reply:
main reason...not enough coffee. i'll change it to make it consistent. now that i see it, it bothers me